index.js

```diff
@@ -0,0 +1,52 @@
+function getRandomIntInclusive(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+// console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+const diceTwo = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+function printDice(diceOneNumber, diceTwoNumber) {
+	let newClass = "img-0"+diceOneNumber;
+	let elm = document.getElementById("dice-01");
+
+	if(elm.className !== newClass){
+		elm.className = newClass;
```

Comment:
Well done on switching the entire class with a conditional and `Element.className`